static/app/components/profiling/threadSelector.tsx

```diff
@@ -34,7 +34,9 @@ function ThreadMenuSelector({
         details: (
           <ThreadLabelDetails
             duration={makeFormatter(profile.unit)(profile.duration)}
-            samples={profile.samples.length}
+            // plus 1 because the last sample always has a weight of 0
+            // and is not included in the raw weights
+            samples={profile.rawWeights.length + 1}
```

Comment:
Not 100% convinced we should make the change in because, for the sampled format, this will mean that the number of samples in the various threads will often be the same. But perhaps that's a positive change since it reflects the number of samples we collected.